src/en-GB/community/events/README.md

```diff
@@ -0,0 +1,76 @@
+<p align="center"><img src="https://i2.wp.com/ceph.io/wp-content/uploads/2016/07/Ceph_Logo_Standard_RGB_120411_fa.png?resize=322%2C148&ssl=1" alt="Ceph" /></p>
+
+# Events
+
+## Frontmatter
+
+```yaml
+---
+title: Event title
+date: yyyy-mm-dd
+end: yyyy-mm-dd
+location: Event location
+venue: Event venue
+map: "https://www.google.com/maps"
+register: "https://www.eventbrite.com/"
+image: "/assets/image.jpg"
+links:
+  videos: "https://www.youtube.com/"
+  slides: "https://ceph.io/event.pdf"
+sponsors:
+  - label: Platinum sponsors
+    list:
+      - name: SoftIron
+        logo: "/assets/image.png"
+        website: "https://softiron.com"
+      - name: RedHat
+        logo: "/assets/image.png"
+        website: "https://redhat.com"
+  - label: Gold sponsors
+    list:
+      - name: SUSE
+        logo: "/assets/image.png"
+        website: "https://suse.com"
+tags:
+  - taxonomy
+  - taxonomy
+---
+
+```
+
+- `title` (String) — Determines the link text as it appears in the event navigation card. Also used as the `<title>` for the page.\*
+- `date` (Date) — Determines the start date of the event.
+- `end` (Date) — Determines the end date of the event.
+- `location` (String) — Determines the location of the event, city and country.
+- `venue` (String) — Determines the venue of the event, building name and street.
+- `map` (String) — Determines the map of the event with a google maps link.\*
```

Comment:
```suggestion
- `map` (String) — Google Maps URL for the event venue location\*
```

Just a convention thing, but wonder if these could be more declaritive/explicit. Might save on repeating "Determines"

Side note: Do you think we could derive the Google Maps link from the Venue + Location information? I.e. `https://www.google.com/maps/search/ibis+Styles+Wroclaw+Centrum` ([Docs](https://developers.google.com/maps/documentation/urls/get-started#search-action) re: escaping characters, etc.)

Reply:
I've revised the `determines` language across all the current READMEs.

I like the idea of inferring the maps link from the venue and location. 

@adamduncan I've implemented it like this, what do you think?

{% set map = venue + '-' + location %}

`https://www.google.com/maps/search/?api=1&query={{ map | slug }}`